Fabric.Authorization.API/RemoteServices/Identity/Providers/IdentityServiceProvider.cs

```diff
@@ -47,23 +49,21 @@ public IdentityServiceProvider(IHttpClientFactory httpClientFactory, IAppConfigu
             var tokenClient = new TokenClient(tokenUriAddress, "fabric-authorization-client", settings.ClientSecret);
             var accessTokenResponse = await tokenClient.RequestClientCredentialsAsync(IdentityScopes.SearchUsersScope).ConfigureAwait(false);
 
-            var httpClient = new HttpClientFactory(
-                tokenUriAddress,
-                "fabric-authorization-client",
-                settings.ClientSecret,
-                null,
-                null).CreateWithAccessToken(new Uri(settings.Authority), accessTokenResponse.AccessToken);
+            var httpRequestMessage = _httpRequestMessageFactory.CreateWithAccessToken(HttpMethod.Post, new Uri("api/users"),
```

Comment:
Do we need to add the full Uri here since we aren't setting a BaseAddress with HttpClient?